### PR TITLE
use of checkers on docker_cli_diff_test.go

### PR DESCRIPTION
**Pull request by echo33**
Signed-off-by: weiyan weiyan3@huawei.com

### CODE REVIEW

File: integration-cli/docker_cli_diff_test.go [hunk 1]
```diff
@@ -22,9 +23,7 @@ func (s *DockerSuite) TestDiffFilenameShownInOutput(c *check.C) {
 			break
 		}
 	}
-	if !found {
-		c.Errorf("couldn't find the new file in docker diff's output: %v", out)
-	}
+	c.Assert(found, checker.Equals, true)
```
Comment (vdemeester) [inline integration-cli/docker_cli_diff_test.go]: There is now `checker.True` and `checker.False` checkers.

File: integration-cli/docker_cli_diff_test.go [hunk 2]
```diff
@@ -78,15 +75,13 @@ func (s *DockerSuite) TestDiffEnsureOnlyKmsgAndPtmx(c *check.C) {
 	}
 
 	for _, line := range strings.Split(out, "\n") {
-		if line != "" && !expected[line] {
-			c.Errorf("%q is shown in the diff but shouldn't", line)
-		}
+		c.Assert(line != "" && !expected[line], checker.Equals, false)
```
Comment (vdemeester) [inline integration-cli/docker_cli_diff_test.go]: Could probably be splitted in 2 asserts (one with `Equals` and one with `True`) to be more readable

```go
c.Assert(line, checker.Equals, "")
c.Assert(expected[line], checker.True)
```

Comment (HuKeping) [inline integration-cli/docker_cli_diff_test.go]: I'm not very sure about this, what about when `line != ""` and ` expected[line]` happen?

Comment (vdemeester) [inline integration-cli/docker_cli_diff_test.go]: @HuKeping oh you're right :sweat_smile:, nevermind my first comment then :stuck_out_tongue_closed_eyes: (although maybe `c.Assert(line == "" || expectedLine, checker.True)` might be more readable).

Comment (HuKeping) [inline integration-cli/docker_cli_diff_test.go]: +1 for adjust it to `true`

**Comment by echo33** [comment]
@vdemeester thanks for your advice, I'll adjust it.

**Comment by vdemeester** [comment]
LGTM 🐸

**Comment by estesp** [comment]
LGTM

**Pull request merged**